gatsby/src/components/ConferenceHero.js

```diff
@@ -76,56 +76,56 @@ const Hero = styled.div`
     margin-top: 10px;
     font-size: 15px;
   }
-  img{
-    height: 60px;
-  }
-}
-@media (max-width: 300px){
-  #hero-box{
-    padding: 2px;
-  }
   #hero-text, #hero-buttons{
     text-align: center;
   }
+  img{
+    height: 60px;
+  }
 }
 `;
 
 function ConferenceHero() {
-
   //query for gatsby-img
+  /* const data = useStaticQuery(graphql`
+    query {
+      imageSharp {
+        fluid(maxWidth: 78) {
+          ...GatsbyImageSharpFluid
+       ; */
```

Comment:
I left this commented out because I was having trouble getting gatsby image fluid to work and wanted to go over it with the group.